src/core/nsfs.js

```diff
@@ -75,13 +76,14 @@ async function main(argv = minimist(process.argv.slice(2))) {
         }
         const http_port = Number(argv.http_port) || 6001;
         const https_port = Number(argv.https_port) || 6443;
+        const backend = (argv.backend);
         const fs_root = argv._[0];
         if (!fs_root) return print_usage();
 
         let fs_config = {
             uid: Number(argv.uid) || process.getuid(),
             gid: Number(argv.gid) || process.getgid(),
-            backend: '',
+            backend: backend || (process.env.GPFS_DL_PATH ? 'GPFS' : ''),
```

Comment:
I wrote in the other thread but now attaching it to the code - the backend setting here is for the account, which is intentionally overridden by the bucket config, which comes from line 128 where we create new NamespaceFS, so we need to remove `backend` from here, and instead put it on new NamespaceFS as `fs_backend` like so:
```js
        const ns_fs = new NamespaceFS({
            fs_backend: argv.backend || (process.env.GPFS_DL_PATH ? 'GPFS' : ''),
            bucket_path: fs_root + '/' + bucket_name,
            bucket_id: '000000000000000000000000',
            namespace_resource_id: undefined,
        });
```

Reply:
notice that I fixed this comment ^ to keep fs_config.backend and use it here, so that we can define the defaults once, in the main function, and then use the actual backend value inside the inner function init_request_sdk.

Reply:
done